app/code/Magento/Catalog/Model/Product/Attribute/Source/StockStatus.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Catalog\Model\Product\Attribute\Source;
+
+use Magento\Eav\Model\Entity\Attribute\Source\AbstractSource;
+use Magento\Eav\Model\Entity\Attribute\Source\SourceInterface;
+use Magento\Framework\Data\OptionSourceInterface;
+
+/**
+ * Product stock status functionality model
+ *
+ * @api
+ * @since 100.0.2
+ */
+class StockStatus extends AbstractSource implements SourceInterface, OptionSourceInterface
+{
+    /**
```

Comment:
Looks like you'd like this phpdoc to be associated with both constants below. Here is a [good example of syntax](https://github.com/magento/magento2/blob/53a896645c8a1a55fe7842ce621a1f79176b6d38/lib/internal/Magento/Framework/UrlInterface.php#L13-L22) on how to add a single phpdoc for several constants.

Reply:
@sivaschenko such syntax is not allowed: https://devdocs.magento.com/guides/v2.2/coding-standards/docblock-standard-general.html#DocBlock-templates